CMakeLists.txt

```diff
@@ -62,6 +62,24 @@ if(moveit_ros_planning_interface_FOUND)
   )
 endif()
 
+if(moveit_ros_planning_interface_FOUND)
```

Comment:
Does the xbox_arm node need moveit? If not, remove the if condition and the dependencies